pysatModels/tests/test_utils_match.py

```diff
@@ -8,8 +8,9 @@
 
 import pysat
 
-import pysatModelUtils as pysat_mu
-import pysatModelUtils.utils.match as match
+import pysatModels as pysat_mu
```

Comment:
Let's change the nickname (since it's not logical anymore).  How about `ps_mod`?